proxies/java/src/main/java/com/dvc_harness/proxy/controller/ProxyController.java

```diff
@@ -21,4 +29,41 @@ public Spec spec() {
         return new Spec();
     }
 
+    @PostMapping("/client")
+    public MessageResponse client(@RequestBody ClientRequestBody body, HttpServletResponse response) {
+        if (body.enableCloudBucketing != null && body.enableCloudBucketing) {
```

Comment:
whats up with this check? Why check if its not null and then check if its truthy?

Reply:
lovely